src/plugins/wikisearch.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Vencord, a modification for Discord's desktop app
+ * Copyright (c) 2023 Vendicated and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import { ApplicationCommandInputType, ApplicationCommandOptionType, findOption, sendBotMessage } from "@api/Commands";
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+
+
+export default definePlugin({
+    name: "Wikisearch",
+    description: "Searches Wikipedia for your requested query. (/wikisearch)",
+    authors: [Devs.Samu],
+    dependencies: ["CommandsAPI"],
+    commands: [
+        {
+            name: "wikisearch",
+            description: "Searches Wikipedia for your request.",
+            inputType: ApplicationCommandInputType.BUILT_IN,
+            options: [
+                {
+                    name: "search",
+                    description: "Word to search for",
+                    type: ApplicationCommandOptionType.STRING,
+                    required: true
+                },
+            ],
+            execute: async (_, ctx) => {
+                const word = findOption(_, "search");
+
+                if (!word) {
+                    return sendBotMessage(ctx.channel.id, {
+                        content: "No word was defined!"
+                    });
+                }
+
+                const data = await fetch(`https://en.wikipedia.org/w/api.php?action=query&format=json&list=search&formatversion=2&origin=*&srsearch=${word}`).then(response => response.json())
+                    .catch(err => {
+                        console.log(err);
+                        return sendBotMessage(ctx.channel.id, { content: "There was an error. Please report this issue on the Vencord repository." });
+                    });
+
+                if (!data.query) {
```

Comment:
```suggestion
                if (!data.query?.search?.length) {
```
then u can just merge this if with the one below